scripts/duplicate_pr.py

```diff
@@ -0,0 +1,124 @@
+from enum import Enum
+
+import argparse
+import sh
+
+
+class BranchName(Enum):
+    MASTER = "master"
+    FORMPLAYER = "formplayer"
+
+
+def get_git(path=None):
+    return sh.git.bake(_tty_out=False, _cwd=path)
+
+
+def get_target_branch(orig_target_branch:str):
+    if orig_target_branch == BranchName.FORMPLAYER.value:
+        return BranchName.MASTER.value
+    elif orig_target_branch == BranchName.MASTER.value:
+        return BranchName.FORMPLAYER.value
+    else:
+        print("Only following branches are allowed: {}".format(", ".join([key.value for key in BranchName])))
+        exit(1)
+
+
+def git_create_branch(orig_branch_name:str, new_branch_name: str, git=None):
+    git = git or get_git()
+    try:
+        git.checkout(orig_branch_name)
+    except sh.ErrorReturnCode_1 as e:
+        print(red(e.stderr.decode()))
+        exit(1)
+    try:
+        git.checkout('-b', new_branch_name)
+    except sh.ErrorReturnCode_128 as e:
+        print(red(e.stderr.decode()))
+        exit(1)
+
+
+def git_fetch_pr_branch(branch_name:str, git=None):
+    git = git or get_git()
+    # fetch remote branch without switching branches
+    input = "{0}:{0}".format(branch_name)
```

Comment:
would this make a string like `branch_name:branch_name` ? If so, what's the purpose of doing so ?

Reply:
`git fetch origin branch_name:branch_name` has the same result as 
`git fetch origin branch_name` + `git checkout branch_name` but without having to switch to the new branch. I think switching to the new branch would be a side affect of this function, so I wanted to avoid that.

Reply:
ahh interesting. We probably don't need to do `git checkout branch_name` here though and just fetching should be enough ?

Reply:
Just fetching doesn't create the branch locally if the branch doesn't already exist. So `get_new_commits` step would throw something like "object does not exist"

Reply:
sounds right, thanks!